<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QPushButton" name="pushButton_updateView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3D">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>560</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_Nz">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>60</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_Nx">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_Ny">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>60</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Nz">
   <property name="geometry">
    <rect>
     <x>1069</x>
     <y>60</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ny">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>60</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ny:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ViewDirector">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> U:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_viewDirectorV">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>90</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>V:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_W">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>90</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>W:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_viewDirectorU">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_viewDirectorW">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savaConfig">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_displayResults">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startSimulation">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>580</y>
     <width>143</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radio_HAADF">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>HAADF</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_HRTEM">
      <property name="text">
       <string>HRTEM</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_titled_5">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: none;
	font-size: 20px;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Microsope Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpts">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="test" extracomment="test"/>
   </property>
   <property name="text">
    <string>gpts</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_gpts_x">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_sampling">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>150</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sampling">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Max.thickness [nm]:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_tds">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>520</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>K,TDS runs:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_temperature">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>520</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_TDS">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>520</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Temperature">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>520</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_TDS">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>520</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TDS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_titled_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: none;
	font-size: 20px;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Detectors</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_outerAngle">
   <property name="geometry">
    <rect>
     <x>997</x>
     <y>490</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>250</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mrad_8">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>490</y>
     <width>32</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>mrad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_OuterAngle">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Outer angle:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_innerAngle">
   <property name="geometry">
    <rect>
     <x>801</x>
     <y>490</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>580</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radio_CPU">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_GPU">
      <property name="text">
       <string>GPU</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>641</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor_file">
    <item>
     <widget class="QLabel" name="label_filename">
      <property name="font">
       <font>
        <italic>false</italic>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>FileName</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="value_filePath"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_openFile">
      <property name="text">
       <string>open File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clearFile">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="value_gpts_y">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpts_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>120</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="test" extracomment="test"/>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearParams">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>590</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ClearParams</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titled_9">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: none;
	font-size: 20px;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Sample Parameters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eV">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>270</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>kV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_highVoltage">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>270</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_HighVoltage">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Accelerating voltage Uacc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Defocus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Defocus</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_defocus">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>330</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-8.7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A_3">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>330</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>nm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_scherzer">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scherzer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Cs5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Spherical Aberr. C3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_sphericalAberrC5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>300</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-0.02</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_um">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>300</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_InnerAngle">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>490</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Inner angle:</string>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>580</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_pixel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>120</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>pixel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>150</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>nm</string>
   </property>
  </widget>
  <widget class="QLabel" name="showImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>641</width>
     <height>485</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>645</width>
     <height>485</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border:2px solid rgb(182, 182, 182);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>580</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="showImage_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border:2px solid rgb(182, 182, 182);
}</string>
   </property>
   <property name="text">
    <string>Crytal Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Project Cell Size</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>View director: </string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ny_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nx:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_gaussian_spread">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>360</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0.08</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gaussian_spread">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gaussian distribution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mrad_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>390</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>mrad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_apertureRadiusAlpha">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>390</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_semiangle_cutoff">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aperture radius alpha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_focal_spread">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Focus spread Delta(1/e half with)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_focal_spread">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>420</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A_4">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>360</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Å</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A_5">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>420</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>[nm]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_thincknessMul">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>150</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setThickness">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SetThickness</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>223</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radio_fullScan">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Full Scan</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_centerScan">
      <property name="text">
       <string>Center Scan</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_filename_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>sampling thickness:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_samplingThickness">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>180</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A_6">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>180</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Å</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
